bundles/extensions/healthcheck/core/src/main/java/org/apache/sling/hc/api/HealthCheck.java

```diff
@@ -66,7 +66,13 @@
      * will be executed asynchronously using the cron expression provided.
      */
     String ASYNC_CRON_EXPRESSION = "hc.async.cronExpression";
-    
+
+    /**
+     * Optional service property: Time To Leave for health check {@link Result}.
+     * The value of this property must be of type {@link Long}.
+     */
+    String TTL = "hc.ttl";
```

Comment:
Can the property be named `hc.resultCacheTtlInMs`? (would be in line with global config and clearly describes it's about the result cache, not about the HC execution time)